outport/process/transactionsfee/transactionsFeeProcessor.go

```diff
@@ -116,15 +121,20 @@ func (tep *transactionsFeeProcessor) prepareTxWithResultsBasedOnLogs(
 	}
 
 	for _, event := range txWithResults.log.GetLogEvents() {
-		identifier := string(event.GetIdentifier())
-		shouldIgnore := identifier != core.SignalErrorOperation && identifier != core.WriteLogIdentifier
-		if shouldIgnore || hasRefund {
-			continue
+		switch {
```

Comment:
I would separate this in a new function so the switch can be eliminated by using early exits

Reply:
refactored